maven-model/src/main/mdo/maven.mdo

```diff
@@ -1087,8 +1087,10 @@
           <version>3.0.0+</version>
           <description>
             <![CDATA[
-            The version of the dependency, e.g. <code>3.2.1</code>. In Maven 2, this can also be
-            specified as a range of versions.
+            The version requirement of the dependency, e.g. <code>3.2.1</code>. The actual version will be resolved based on the usage context.
+            Version requirement can also be specified as a range of versions, e.g. <code>[3.2.0,)</code>. This is discouraged as it breaks <i>predictability</i> of resolved version.
```

Comment:
```suggestion
            Version requirement can also be specified as a range of versions, e.g. <code>[3.2.0,)</code>. User have to be aware that range versions may break <i>predictability</i> of resolved version in some cases.
```

Reply:
too long: going back to "as it may break predictability"